EnrollmentCollectionEntity.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+import { Entity } from 'siren-sdk/es6/Entity.js';
+import { Rels } from 'd2l-hypermedia-constants';
+
+export class EnrollmentCollectionEntity extends Entity {
+	enrollmentsHref() {
+		if (!this._entity || !this._entity.getSubEntitiesByRel) {
+			return;
+		}
+		const enrollmentEntities = this._entity.getSubEntitiesByRel(Rels.userEnrollment);
+		return enrollmentEntities.map(e => e.href || undefined).filter(href => href);
```

Comment:
Minor: I think you can remove the `|| undefined` - If `e.href` is falsy then it will get removed in the `filter`
```suggestion
		return enrollmentEntities.map(e => e.href).filter(href => href);
```